test/e2e/llmisvc/test_llm_autoscaling_direct_keda.py

```
@@ -0,0 +1,231 @@
+def _create_and_wait(kserve_client, test_case):
+    create_llmisvc(kserve_client, test_case.llm_service)
+    wait_for_llm_isvc_ready(
+        kserve_client, test_case.llm_service, test_case.wait_timeout
+    )
+
+
+def _cleanup(kserve_client, test_case):
+    if os.getenv("SKIP_RESOURCE_DELETION", "False").lower() in ("false", "0", "f"):
+        try:
+            delete_llmisvc(kserve_client, test_case.llm_service)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)